packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -487,6 +477,21 @@ Router = Router.extend({
     return false;
   },
 
+  createReactViewHandler(routeName, additionalParams) {
```

Comment:
Can we call this function [here and remove the duplicate lines](https://github.com/mozilla/fxa/blob/a74024c1fc4eff5c329f15a0560dd97a4af30249/packages/fxa-content-server/app/scripts/lib/router.js#L504)?

Reply:
SMART! 🧠